apps/docs-snippets/src/guide/contracts/minted-token-asset-id.test.ts

```diff
@@ -30,4 +30,16 @@ describe(__filename, () => {
     expect(mintedAssetId).toBeDefined();
     expect(txResult.transactionResult.isStatusSuccess).toBeTruthy();
   });
+
+  it('should create valid asset ID', async () => {
+    // #region create-asset-id-1
+    const subID = '0xc7fd1d987ada439fc085cfa3c49416cf2b504ac50151e3c2335d60595cb90745';
+
+    const assetId = createAssetId(contract.id.toB256(), subID);
+    const { value } = await contract.functions.echo_asset_id_comparison(assetId).simulate();
+    // #endregion create-asset-id-1
```

Comment:
```suggestion
    // #endregion create-asset-id-1
    
    const { value } = await contract.functions.echo_asset_id_comparison(assetId).simulate();
```

I think we move the contract call outside the snippet, I dont think it adds value to the snippet.

Reply:
I can do that, although I believe having it as part of the snippet shows how the Asset ID returned can be used?

If that's not necessary, I'll replace it in one more commit.

Reply:
But as we are not showing the contract is in the documentation, we do not have any context as to what function `echo_asset_id_comparison` is performing, or what the `value` is. I agree it _could_ be useful, but without the context it only adds complexity to the snippet rather than information. I appreciate the [previous snippet](https://github.com/FuelLabs/fuels-ts/blob/08aea1d7ac3406ab219e26ff315fa8469df4f8a6/apps/docs-snippets/src/guide/contracts/minted-token-asset-id.test.ts#L25) calls `mint_coins` but I think that is more self describing, especially as the function it is calling is getting the asset ID for the minted coin.

Reply:
Alright, that makes sense. I've pushed a new commit